lib/core/include/irods/base64.h

```diff
@@ -11,17 +11,12 @@
 
 #include "irods/rodsDef.h"
```

Comment:
This doesn't look like it is needed.

Try removing it and seeing if things still compile. If it fails to compile, put this `#include` in the cpp file.

Reply:
done

Reply:
done

Reply:
I still see the `#include`...